app/jobs/resolution_proofing_job.rb

```diff
@@ -25,6 +25,15 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
 
     applicant_pii = decrypted_args[:applicant_pii]
 
+    threatmetrix_result = nil
+    if use_lexisnexis_ddp_threatmetrix_before_rdp_instant_verify?
+      user = User.find_by(id: user_id)
```

Comment:
so I see that we're passing `user_id`, but I think that ActiveJob will correctly serialize/deserialize ActiveRecord models, we should just pass `user` if we wanted to I think

Reply:
I modeled it after AddressProofingJob which takes user_id